vm.go

```diff
@@ -103,6 +103,8 @@ type ioSession struct {
 	// started. This is used to stop the shim from sending stdin data to a
 	// non-existent process.
 	processStarted chan interface{}
+
+	state tokenState
```

Comment:
It would be good to put a comment on this, for consistency with the other fields.